src/components/cylc/Task.vue

```diff
@@ -160,28 +159,27 @@
     }
 </style>
 
-
 <script>
-    export default {
-        name: 'Task',
-        props: {
-            status: {
-                type: String,
-                required: true
-            },
-            progress: {
-                type: Number,
-                required: false,
-                default: 0
-            }
-        },
-        computed: {
-            progressStyle: function () {
-                return {
-                    'stroke-dashoffset':
-                        `calc(157 - (157 * ${this.progress/100} + ))`
-                }
-            }
-        }
+export default {
+  name: 'Task',
+  props: {
+    status: {
+      type: String,
+      required: true
+    },
+    progress: {
+      type: Number,
+      required: false,
+      default: 0
+    }
+  },
+  computed: {
+    progressStyle: function () {
+      return {
+        'stroke-dashoffset':
+                        `calc(157 - (157 * ${this.progress / 100} + ))`
```

Comment:
Is this supposed to have so much indentation? (Or just bad Github diff?)

Reply:
This change was done automatically by `npm run lint`. Interesting that it seems to try to keep the line under 120 characters I think (there is no limit in the linter I believe). Fixed.